backup/README.md

```diff
@@ -91,7 +85,13 @@ To dump out the latest copy of the database to recover from:
 
 You will then need to clone the contents of the `barman_recover` volume into a suitable place for the Postgres server to read from.
 
-To remove all traces of barman (the cron job, the container and the volumes), use:
+There is a nightly recovery - you can trigger this at other times by running
```

Comment:
This line won't mean much to anyone who hasn't already grokked the system. Perhaps: "Every night a snapshot of the database is written out to the `barman_nightly` volume. You can trigger this at other times..."

Reply:
Also, maybe a note to say something like: But if you want a manual dump, try using the `recover` command and the `barman_recover` volume?